mlir/lib/ExecutionEngine/ROCm/CMakeLists.txt

```diff
@@ -67,15 +67,6 @@ set_source_files_properties(BackendUtils.cpp PROPERTIES
   COMPILE_OPTIONS "-Wno-gnu-anonymous-struct;-Wno-nested-anon-types")
 
 set(LIBS
-  lldCommon
-  lldDriver
```

Comment:
I was having trouble with compilations failing because they couldn't find liblldDriver.so while making libLLVMROCmBackendUtils.so.  I think it may be a dependence problem, but the nearest upstream equivalent is mlir-vulkan-driver which mentions avoiding depending on LLVM libraries.  Removing these lines solved my problem and hasn't caused others.